our_land/sql/individualervenlandownershipinhectaresbygender.sql

```diff
@@ -0,0 +1,103 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 9.5.12
+-- Dumped by pg_dump version 9.5.12
+
+-- Started on 2018-08-23 11:54:41 EAT
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+
```

Comment:
Ensure **ALL** sql files have:

 + `DROP INDEX IF EXISTS  ...;`
 + `ALTER TABLE .... DROP CONSTRAINT IF EXISTS ...;`
 + `DROP SEQUENCE IF EXISTS ...;`
 + `DROP TABLE IF EXISTS ...;`

If  the sql file creates indexes, constraints, sequences and/or tables. This makes sure the migrations are repeatable.

PS: I think `--clean` option of `pg_dump` does add these statements.